src/pages/designsystem.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import DesignSystem from '../components/DesignSytem';
```

Comment:
1. spell error should `DesignSystem`
2. this file's name should `design-system`